### ISSUE TITLE
Error from a music XML file - Spring can...

### ISSUE THREAD

**Issue by AlbertHart** [opensheetmusicdisplay/opensheetmusicdisplay#701]
I was having trouble on my machine loading this .musicxml file. (Attached in ZIP)
I had exported it from MuseScore. It is in the ZIP file below.

Both my program and the Demo get this error repeatedy:

demo.min.js:1 Error: <path> attribute d: Expected number, "MNaN 1744MNaN 173…".

If you can help identify the problem in the MusicXML file, I can probably pre-process the MusicXML file to patch it.

![image](https://user-images.githubusercontent.com/8965352/77837163-2febd480-7123-11ea-8821-1761f2fd9925.png)

[Spring_Can_Really_Hang_You_Up_the_Most.musicxml.zip](https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/files/4398133/Spring_Can_Really_Hang_You_Up_the_Most.musicxml.zip)

**Comment by fredmeister77** [comment]
@AlbertHart , Can you provide your browser & platform information? This musicxml file is succeeding for me on Firefox 68.6.0 (Debian) and Chrome 80.0.3987.149 (Debian), and I need to be able to reproduce on my end.

Thanks!

**Comment by sschmidTU** [comment]
This is from the [official demo](https://opensheetmusicdisplay.github.io/demo/), which runs on the current release.
This would probably need to be run with debugging, because demo.js:1 doesn't tell much about where this is coming from. We probably generate NaN somewhere and hand it to Vexflow, which rightfully complains. Or maybe the error is in Vexflow.

**Comment by AlbertHart** [comment]
I don't have the music xml libraries in debug mode.

So I just deleted things until the error messages went away.

It turns out that the words entry in measure 25 messed things up. When I remove it, things work OK.



`<words default-y="26.50" relative-y="20.00">&lt;b&gt;&lt;font size=&quot;13&quot;&gt;&lt;/font&gt;TO CODA 2ND TIME&lt;/b&gt;</words>`

In particular, when I remove the text: &lt;b&gt;&lt;font size=&quot;13&quot;&gt;&lt;/font&gt;TO CODA 2ND TIME&lt;/b&gt;

It works OK:

OK:

          <words>&lt;b&gt;&lt;&gt;&lt;/font&gt;TO CODA 2ND TIME&lt;/b&gt;</words>

**Comment by sschmidTU** [comment]
I've tried debugging this issue, but during step by step debugging, it doesn't happen.
What i've found out is that this seems to happen during MusicSheetDrawer.drawPage(), specifically this.drawMusicSystem().
In my case this was system 17, bar 31, where the coda sign is.
So, it seems related to the coda sign, but i can't locate it during debugging.
I've checked all kinds of numbers whether they're NaN (using isNaN()), but to no avail.
